README.md

```diff
@@ -18,3 +23,69 @@ This work is licensed under GNU GENERAL PUBLIC LICENSE Version 3
 
 # CODE OF CONDUCT:
 You can find the Twitter Community Code of Conduct as a sample in code_of_conduct.md
+
+# AUFGABENSTELLUNG:
+Die folgende Aufgabenstellung ist aus der Plattform Olat �bernommen
+
+Project Infrastructure
```

Comment:
Diese Zwischenüberschriften sollten mit zwei Hashes eingeleitet und von einer Leerzeile gefolgt werden. Also statt:

    Project Infrastructure
    Create the project’s infrastructure:

Besser:

    ## Project Infrastructure

    Create the project’s infrastructure: